internal/cmd/clone/clone.go

```diff
@@ -19,14 +23,37 @@ func (o *options) AddFlags(cmd *cobra.Command) {
 		"",
 		"specify branch to check out",
 	)
+	cmd.Flags().StringSliceVarP(
+		&o.expectedRootKeys,
+		"root-keys",
```

Comment:
```suggestion
		"root-key",
```

note that you'd pass in `--root-key <key> --root-key <key>` rather than `--root-keys <key> <key>`.

Reply:
Fixed